OverlayPlugin.Core/Overlays/LabelOverlay/LabelOverlayConfigPanel.cs

```diff
@@ -175,6 +158,7 @@ private void textBoxGlobalHotkey_KeyDown(object sender, KeyEventArgs e)
 
         private void checkLock_CheckedChanged(object sender, EventArgs e)
         {
+            if (this.config.Url == "" || this.config.Text == "") return;
```

Comment:
Checkbox config should can be changed even there's no URL or text specified, please fix this.